docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -487,18 +487,16 @@ Vars plugins
 
 Vars plugins inject additional variable data into Ansible runs that did not come from an inventory source, playbook, or command line. Playbook constructs like 'host_vars' and 'group_vars' work using vars plugins.
 
-Vars plugins were partially implemented in Ansible 2.0 and rewritten to be fully implemented starting with Ansible 2.4. Vars plugins are supported by collections starting with Ansible 2.10.
+Vars plugins were partially implemented in Ansible 2.0 and rewritten to be fully implemented starting with Ansible 2.4. Vars plugins are supported by collections starting with Ansible 2.10. Ansible 2.0 did not pass passwords to older plugins, so vaults were unavailable.
 
-Older plugins used a ``run`` method as their main body/work:
+Legacy plugins used a ``run`` method as their main body/work:
 
 .. code-block:: python
 
     def run(self, name, vault_password=None):
         pass # your code goes here
 
-
-Ansible 2.0 did not pass passwords to older plugins, so vaults were unavailable.
-Most of the work now  happens in the ``get_vars`` method which is called from the VariableManager when needed.
```

Comment:
Can you remove the refactoring of the existing documentation? I'm already working on this in https://github.com/ansible/ansible/pull/79945, and some of the legacy documentation is going to go away soon.